### PR TITLE
Various Updates

### PR DESCRIPTION
**Pull request by yuce**
- Use pause consistently
- Updated doc for MacOS
- Do not print an error message on user cancelled

### CODE REVIEW

File: docs/modules/ROOT/pages/install-clc.adoc [hunk 1]
```diff
@@ -30,6 +36,12 @@ ZIP (Apple Silicon)::
 +
 . Go to the https://github.com/hazelcast/hazelcast-commandline-client/releases/latest[latest release page], and locate the ARM compressed file (`hazelcast-clc_v{full-version}_darwin_arm64.zip`).
 . Download and unzip the file.
+. Remove the `clc` binary from quarantine if you get a security warning when running it:
++
+[source,shell,subs="attributes"]
+----
+xattr -d com.apple.quarantine hazelcast-clc_v{full-version}_darwin_amd64/clc
```
Comment (kutluhanmetin) [inline docs/modules/ROOT/pages/install-clc.adoc:43]: I think this should be arm, right?

File: clc/cmd/exec_context.go [hunk 2]
```diff
@@ -234,27 +234,26 @@ func (ec *ExecContext) WrapResult(f func() error) error {
...
-				msg = fmt.Sprintf("Error: %s", errStr)
-			}
-			if ec.Interactive() {
-				I2(fmt.Fprintln(ec.stderr, color.RedString(msg)))
-			} else {
-				I2(fmt.Fprintln(ec.stderr, msg))
-			}
+			errStr = err.Error()
+		}
+		if verbose {
```
Comment (kutluhanmetin) [inline clc/cmd/exec_context.go:248]: I think this is optional: we can get rid of else statements by initializing them with the values inside `else`s.

Comment (yuce) [reply clc/cmd/exec_context.go:248]: I am also not a fan of the `else`s, but in this case that would require calling functions more than once.

File: docs/modules/ROOT/pages/install-clc.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/ROOT/pages/install-clc.adoc]: Thanks, updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/245/commits/8aa04515f273aeca9661a01a90277b98ba1fcc0f

**Pull request merged**